app/views/refinery/inquiries/inquiry_mailer/notification.text.erb

```diff
@@ -5,6 +5,9 @@
 <%=raw t('.inquiry_starts') %>
 
 <%=raw t('.from') %>: <%= @inquiry.name %>
+<% if Refinery::Inquiries.show_company_field == true %>
+<%=raw t('.company') %>: <%= @inquiry.company %>
+<% end %>
```

Comment:
does this conditional now leave a blank line in the email template? If so, I suspect it could be avoided with:

```ruby
<%=raw "#{t('.company')}: #{@inquiry.company}" if Refinery::Inquiries.show_company_field == true -%> 
```

Reply:
Not even... I was expecting the same behavior as you mention so I quickly tried it out by setting `config.show_company_field = false` and the result is that there are no extra blank line between `From:` and `Email:`.

Reply:
Neat!